File: python/suma-cyfr.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  suma-cyfr.py
#  
#  Copyright 2018  <>
#  
#  
#  
#  


def main(args):
    n = int(input("podaj pierwszą liczbe: "))
    
    while n < 10:
        print("błąd")
        n = int(input("podaj pierwszą liczbe: "))
        
    suma = 0
    while n > 0:
        suma = suma + (n % 10)
        n = int(n / 10)
    print("suma:", suma)
        
    
    
    
    
    
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: cpp/suma-cyfr.cpp
/*
 * suma-cyfr.cpp
 */


#include <iostream>
using namespace std;

int main(int argc, char **argv)
{
	int a = 0;
    while (a < 10) {
        cout << "podaj liczbę ";
        cin >> a;
    };
    int suma = 0;
    while (a > 0) {
        suma = suma + (a % 10);
        a = a / 10;
    };
    cout << "suma " << suma;
    
	return 0;
}

